src/main/java/com/salesforce/storm/spout/sideline/handler/SidelineSpoutHandler.java

```diff
@@ -294,11 +316,15 @@ public void stopSidelining(SidelineRequest sidelineRequest) {
         // assigned to this particular sideline spout instance
         for (final ConsumerPartition consumerPartition : endingState.getConsumerPartitions()) {
             // This is the state that the VirtualSpout should start with
-            final SidelinePayload sidelinePayload = getPersistenceAdapter().retrieveSidelineRequest(
+            final SidelinePayload sidelinePayload = retrieveSidelinePayload(
                 id,
                 consumerPartition.partition()
             );
 
+            if (sidelinePayload == null) {
```

Comment:
Do we think this should evoke some log msg?  Should we clear/remove the sideline from persisted state?

Reply:
Done